src/lib/components/Treebank/Annotators/Annotators.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import TreebankContext from '../treebank-context';
+
+import Annotators from '../../Annotators';
+
+const AnnotatorsWithContext = () => (
+  <TreebankContext.Consumer>
+    {({ json }) => <Annotators json={json} />}
+  </TreebankContext.Consumer>
+);
+
+export default AnnotatorsWithContext;
```

Comment:
This file wraps the `Annotators` component in the Treebank context. This allows the code to be written like this:

```jsx
<Treebank treebank={treebank}>
  <Annotators />
  <Graph />
  <SomeOtherComponent />
</Treebank>
```

instead of like this:

```jsx
<Annotators treebank={treebank} />
<Graph treebank={treebank} />
<SomeOtherComponent treebank={treebank} />
```